core/geo-json/src/test/java/org/eclipse/sensinact/gateway/geojson/GeoJsonTest.java

```diff
@@ -161,17 +163,25 @@ void testNaN() throws Exception {
                 new TypeReference<List<Geometry>>() {
                 });
 
-        assertEquals(2, geometries.size());
```

Comment:
If we're actually saying that we should be strict then isn't `NaN` in these JSON files invalid. Shouldn't we fail to parse anything?

Reply:
Then we should add an additional check in the CoordinatesDeserializer and throw an exception even when one of the coordinates is NaN, right?